src/certificate.js

```diff
@@ -378,18 +389,30 @@ export default class Certificate {
       return;
     }
 
-    let readableAction = getVerboseMessage(step);
-    log(readableAction);
-    this._updateStatusCallback(step, readableAction, Status.starting);
+    let readableAction;
+    if (step) {
+      readableAction = SUB_STEPS.language[step].actionLabel;
```

Comment:
I think you could still maintain a function around that, but it's no biggy

Reply:
This will be better done once `doAction` & `doSyncAction` are merged into once single function running steps functions dynamically.